docs/tutorials/python_trestle_setup.md

```diff
@@ -0,0 +1,104 @@
+# Tutorial: Install trestle in a python virtual environment
+
+Here are step by step instructions to install trestle.
+
+## *Objective*
+
+Install trestle for manipulation of OSCAL documents.
+
+There are 3 short steps shown below before you can use trestle.
+* Make sure you have a working and recent Python environment
+* Set up a Python virtual environment
+* Download and install trestle
+* Confirm it is working properly
+* Create a trestle workspace
+
+## *Confirm you have python installed*
+
+
+* Ensure you have a modern [Python](https://www.python.org/downloads/) (3.7, 3.8, 3.9).
+
+```
+$ python -V</code>
+Python 3.8.3</code>
+```
+
+## *Setup a virtual environment*
+
+```
+$ cd
+$ python -m venv venv.trestle
+$ source venv.trestle/bin/activate
+(venv.trestle)$
+
+```
+* Ensure you have a modern [pip](https://pip.pypa.io/en/stable/installing/) (19.x or greater).
+
+```
+(venv.trestle)$ pip --version
+pip 19.2.3 from /home...
+```
+## *Install trestle*
+
+* Install [compliance-trestle](https://github.com/IBM/compliance-trestle/blob/develop/README.md).
```

Comment:
Replace with appropriate URL on the ibm.github.io/compliance-trestle website.

Reply:
This was in the itanium files also so I changed there, too.